doc/src/site/sphinx/migration_guide.rst

```diff
@@ -210,6 +210,9 @@ From 3.28.0 to 3.28.1
   The ``H2O_DRIVER_JAR`` should contain a path to a plain H2O driver jar without any extensions.
   For more details, see :ref:`external-backend`.
 
+- The location of Sparkling Water assembly JAR has changed inside the Sparkling Water distribution archive which you
+  download from our download page. The folder has been changed from ``assembly/build/libs`` to just ``jars``.
```

Comment:
`our download page. ` link to download page.?

Reply:
👍